scripts/funcotator/testing/testFuncotator.sh

```diff
@@ -56,6 +57,7 @@ MANUAL_MODE=false
 ################################################################################
 
 # Change this to point to your funcotator data sources folder:
+DATA_SOURCES_PATH_16=/Users/jonn/Development/funcotator_dataSources.v1.6.20190124s
 DATA_SOURCES_PATH=/Users/jonn/Development/funcotator_dataSources_latest
 DATA_SOURCES_PATH_GERMLINE=/Users/jonn/Development/funcotator_dataSources_germline_latest
```

Comment:
Perhaps the base directory should be a script argument instead of hardcoded to your home directory? Is this meant to be used by anyone other than yourself?

Reply:
This is in here so that I can have a consistent script to test with.  Way back when I originally checked it in, Lee suggested that I add in the comment above.  The datasource paths I have in here are long and it's a little painful to type everything in correctly each time.

No one other than myself has ever run this, but it is a useful script.

I can add in warning messages if the paths / files don't exist.  @droazen What do you think?

Reply:
OK - fixed!